samples/Basic/Basic.WebSvc/Controllers/QueryController.cs

```diff
@@ -37,7 +38,149 @@ public Task<IHttpActionResult> GetMetadata(string application, string service)
 		[Route("query/{application}/{service}/{collection}")]
 		public Task<IHttpActionResult> Query(string application, string service, string collection)
 		{
-			return base.QueryAsync(application, service, collection);
+			return QueryAsync(application, service, collection);
+		}
+
+
+		/// <summary>
+		/// Deletes appropriate key & corresponding value from the given reliable collection in the queryable service.
+		/// SINGLE DELETE:
+		/// - DELETE /query/BasicApp/ProductSvc/products and in Body provide a Json:
```

Comment:
Using InBody method is necessary as key can be anything(nested objects) and of any length since it is user defined.